bpf/lib/policy.h

```diff
@@ -48,10 +48,14 @@ static inline bool identity_is_reserved(__u32 identity)
 	return identity < HEALTH_ID;
 }
 
+#ifndef HAVE_L4_POLICY
+#define HAVE_L4_POLICY 0
```

Comment:
Why not have this in `pkg/endpoint/bpf.go`? Not sure if this is a dumb question or not, as BPF-land is still a bit foreign to me.

Reply:
Good idea. I'll address this with any other feedback I might get.